### ISSUE TITLE
Installation Failure

### ISSUE THREAD

**Issue by Janaka-Steph** [vulpemventures/tdex-cli#9]
```
$ npm i -g tdex-cli 
npm WARN deprecated axios@0.19.2: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm ERR! code 1
npm ERR! Command failed: git checkout beta
npm ERR! error: pathspec 'beta' did not match any file(s) known to git
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/steph/.npm/_logs/2021-05-25T18_16_55_517Z-debug.log
```

```
42 silly fetchPackageMetaData error for tdex-protobuf@github:tdex-network/tdex-protobuf#beta Command failed: git checkout beta
42 silly fetchPackageMetaData error: pathspec 'beta' did not match any file(s) known to git
43 timing stage:rollbackFailedOptional Completed in 1ms
44 timing stage:runTopLevelLifecycles Completed in 2151ms
45 verbose stack Error: Command failed: git checkout beta
45 verbose stack error: pathspec 'beta' did not match any file(s) known to git
45 verbose stack
45 verbose stack     at ChildProcess.exithandler (child_process.js:308:12)
45 verbose stack     at ChildProcess.emit (events.js:315:20)
45 verbose stack     at maybeClose (internal/child_process.js:1048:16)
45 verbose stack     at Socket.<anonymous> (internal/child_process.js:439:11)
45 verbose stack     at Socket.emit (events.js:315:20)
45 verbose stack     at Pipe.<anonymous> (net.js:673:12)
```

**Comment by tiero** [comment]
cc @louisinger

**Comment by Janaka-Steph** [comment]
It seems that the same error appeared again

```
$ yarn global add tdex-cli
yarn global v1.22.15
[1/4] 🔍  Resolving packages...
warning tdex-cli > axios@0.19.2: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
error Couldn't find match for "beta" in "refs/heads/dependabot/npm_and_yarn/set-getter-0.1.1,refs/heads/dependabot/npm_and_yarn/striptags-3.2.0,refs/heads/dependabot/npm_and_yarn/tar-4.4.19,refs/heads/develop,refs/heads/master,refs/pull/1/head,refs/pull/10/head,refs/pull/11/head,refs/pull/12/head,refs/pull/18/head,refs/pull/19/head,refs/pull/2/head,refs/pull/20/head,refs/pull/21/head,refs/pull/22/head,refs/pull/23/head,refs/pull/24/head,refs/pull/25/head,refs/pull/26/head,refs/pull/27/head,refs/pull/28/head,refs/pull/29/head,refs/pull/3/head,refs/pull/30/head,refs/pull/31/head,refs/pull/32/head,refs/pull/33/head,refs/pull/34/head,refs/pull/35/head,refs/pull/36/head,refs/pull/37/head,refs/pull/38/head,refs/pull/39/head,refs/pull/4/head,refs/pull/40/head,refs/pull/41/head,refs/pull/41/merge,refs/pull/42/head,refs/pull/42/merge,refs/pull/43/head,refs/pull/44/head,refs/pull/45/head,refs/pull/45/merge,refs/pull/46/head,refs/pull/47/head,refs/pull/47/merge,refs/pull/48/head,refs/pull/5/head,refs/pull/6/head,refs/pull/8/head,refs/pull/9/head" for "https://github.com/tdex-network/tdex-protobuf.git".
```
